File: IdeaProjects/Tienda/src/Productos/Electrodomestico.java
package Productos;

import java.time.LocalDate;

public class Electrodomestico extends Producto {
    double watts;
    String color, descripcion;

    public Electrodomestico(String name, double precio, int stock, int no, LocalDate fechaImportacion, String tipo, double watts,
                            String color, String descripcion) {
        super(name, precio, stock, no, fechaImportacion, tipo);
        this.watts = watts;
        this.descripcion = descripcion;
        this.color = color;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public double getWatts() {
        return watts;
    }

    public void setWatts(double watts) {
        this.watts = watts;
    }
}

File: IdeaProjects/Tienda_py/.idea/Productos/Electrodomestico.py
from datetime import date

class Producto:
    def _init_(self, nombre, precio, stock, numero, fecha_importacion, tipo):
        self.nombre = nombre
        self.precio = precio
        self.stock = stock
        self.numero = numero
        self.fecha_importacion = fecha_importacion
        self.tipo = tipo

class Electrodomestico(Producto):
    def _init_(self, nombre, precio, stock, numero, fecha_importacion, tipo, watts, color, descripcion):
        super()._init_(nombre, precio, stock, numero, fecha_importacion, tipo)
        self.watts = watts
        self.color = color
        self.descripcion = descripcion

    def get_descripcion(self):
        return self.descripcion

    def set_descripcion(self, descripcion):
        self.descripcion = descripcion

    def get_color(self):
        return self.color

    def set_color(self, color):
        self.color = color

    def get_watts(self):
        return self.watts

    def set_watts(self, watts):
        self.watts = watts
